common/components/org.wso2.carbon.data.agents.log4j/src/main/java/org/wso2/carbon/data/agents/log4j/appender/LogEventAppender.java

```diff
@@ -51,117 +49,154 @@
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.*;
```

Comment:
Hi Tishan,

This is fixed in by the commits in https://github.com/wso2/shared-analytics/pull/126

Thanks,
SajithD